pkg/client/cache/reflector.go

```diff
@@ -265,6 +265,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			switch err {
 			case io.EOF:
 				// watch closed normally
+				glog.V(1).Infof("%s: Watch for %v closed normally (with EOF)", r.name, r.expectedType)
```

Comment:
Why does this happen?  We do a full refetch whenever it happens, so hopefully it doesn't happen _too_ often :-)